demo/src/routes/menu/Theme.svelte

```diff
@@ -79,7 +80,9 @@
 			data-bs-toggle="dropdown"
 			aria-expanded={$open$}
 		>
-			<Svg svg={$currentTheme$.icon} />
```

Comment:
Maybe you can remove `icon` from the `currentTheme$` then, as it is not used anymore

Reply:
Indeed, maybe the currentTheme$ should just be a ```writable(string)```,  and would be the id of the current theme.